.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for helping us make a better snapd!
+
+Have you signed the [license
+agreement](https://www.ubuntu.com/legal/contributors) and read the [contribution
+guide](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)?
+
+Please make sure that the following items have been verified: (please leave a
+note if one of those is not applicable):
+
+- [ ] there are unit tests covering your change
+- [ ] unit tests are passing (./run-checks --unit)
+- [ ] static files tests are passing (./run-checks --static)
+- [ ] snapd snap build works with latest snapcraft (snapcraft --verbose)
+- [ ] the change needs an entry in the NEWS.md file
```

Comment:
During the last few iterations I concluded this is not useful.

(1) We need 100% certainty that nothing is missed, which means I need do what I do and run script to detect all PR's between previous ad current release branch, minus cherry-picked items. From this, test only development as well as internal changes are removed. Then related items (e.g. interface changes) are grouped. Then PR's related to same topic (e.g. components) are summarised. I think this is not possible to get right with run-time news updates.

(2) In addition, having entries in NEWS.md for upcoming release means merging back from release branch is not super trivial as it is without.